src/Inventory.js

```diff
@@ -84,7 +84,7 @@ class Inventory extends Component {
           <ul>
             { 
               displayCategories.map((category) => {
-                return <InventoryCat category={category} key={category} products={this.props.allInventory} toggle={this.toggle} toggleState={this.state.toggle} />
+                return <InventoryCat category={category} key={category} products={this.props.allInventory} toggle={this.toggle} toggleState={this.state.toggle} changeCart={this.props.changeCart} />
```

Comment:
Whenever you have a component with a lot of props, it's a good idea to break each prop up, line by line, for readability. Most linters have built in rules that default to having no more than 1 prop per line because of this:
```jsx
<InventoryCat 
    category={category}
    key={category} 
    products={this.props.allInventory} 
    toggle={this.toggle} 
    toggleState={this.state.toggle} 
    changeCart={this.props.changeCart}
/>
```